tools/nodetool

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
```

Comment:
This silly "coding" line is:
1. A Python-only thing
2. Even in Python it was made the default more than a decade ago.
Maybe just delete it? Did you add it yourself or your editor added it?

Reply:
not sure if this is "silly", but AFAICT, this line is not a python-only thing, and it can be consumed by Emacs to initialize file local variable values. see https://www.gnu.org/software/emacs/manual/html_node/emacs/Specifying-File-Variables.html .

Reply:
This is copy-pasta indeed. Will remove.

Reply:
@tchaikov you're right that it started as an Emacs thing, but Python 2 later adopted it as a way to tell Python that the source code itself is in UTF-8. I don't believe bash ever did anything with this comment (to be honest, I don't think bash even cares about the encoding of the script?). Anyway, both Python and Emacs default to utf-8 for many years now, so this comment while it can't hurt (it's a comment...) doesn't help either. Add to this the fact that this script is 100% ASCII, we really don't need this comment :-)